xrpl/wallet/main.py

```diff
@@ -16,69 +17,131 @@ class Wallet:
     details.
     """
 
+    @property
+    def classic_address(self: Wallet) -> str:
+        """
+        The address that publicly identifies this wallet,
+        as a base58 string.
+        """  # noqa: DAR201
```

Comment:
Is this noqa still necessary?

Reply:
Because it is like a function, it wanted the docstring to have a return parameter explanation, but I wanted the docstring to appear alike to other properties (like self.seed in the constructor).